container/usecase.go

```diff
@@ -0,0 +1,21 @@
+package container
+
+import (
+	"context"
+
+	"github.com/utahta/momoclo-channel/usecase"
+)
+
+type usecaseContainer struct {
+	ctx  context.Context
+	repo *repositoryContainer
+}
+
+// Usecase returns use case container
+func Usecase(ctx context.Context) *usecaseContainer {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported func Usecase returns unexported type *container.usecaseContainer, which can be annoying to use